Go/Go.sublime-syntax

```diff
@@ -70,19 +71,23 @@ variables:
   keyword: \b(?:break|case|chan|const|continue|default|defer|else|fallthrough|for|func|go|goto|if|import|interface|map|package|range|return|select|struct|switch|type|var)\b
 
   # https://golang.org/ref/spec#Predeclared_identifiers
-  predeclared_type: \b(?:bool|byte|complex64|complex128|error|float32|float64|int|int8|int16|int32|int64|rune|string|uint|uint8|uint16|uint32|uint64|uintptr)\b
+  predeclared_type: \b(?:any|bool|byte|comparable|complex64|complex128|error|float32|float64|int|int8|int16|int32|int64|rune|string|uint|uint8|uint16|uint32|uint64|uintptr)\b
 
   # https://golang.org/ref/spec#Predeclared_identifiers
   predeclared_func: \b(?:append|cap|close|complex|copy|delete|imag|len|make|new|panic|print|println|real|recover)\b
 
+  ident_blank: \b_\b
+
   # Note: this matches ALL valid identifiers, including predeclared constants,
   # functions and types.
   ident: \b(?!{{keyword}})[[:alpha:]_][[:alnum:]_]*\b
 
+  keyword_or_ident: \b[[:alpha:]_][[:alnum:]_]*\b
```

Comment:
May probably want to use it in `ident`?

```yaml
ident: (?!{{keyword}}){{keyword_or_ident}}
```

Reply:
Wasn't sure if this would work properly because unlike the previous version, this inserts `\b` into the middle of the regex. But tests pass so you might be right.